src/js/components/lazy-load.js

```diff
@@ -62,38 +127,54 @@ class LazyLoad {
     image.removeAttribute('data-lazy')
 
     if (placeholder.getAttribute('data-srcset')) {
-      this.parseBreakpoints(image, placeholder.getAttribute('data-srcset'))
-
-      return image
+      this.checkBreakpointsSource(image, placeholder.getAttribute('data-srcset'))
+    } else {
+      image.src = placeholder.getAttribute('data-src')
     }
 
-    image.src = placeholder.getAttribute('data-src')
-
     return image
   }
 
+  /**
+   * Parse all data-attr from [data-lazy] to <image />
```

Comment:
`Check the list of attributes passed and add to the image element only those with a name different from data-lazy, data-srcset, and data-src.`